UnityWeld/Binding/AbstractMemberBinding.cs

```diff
@@ -25,7 +24,7 @@ public virtual void Init()
         {
             _isInitCalled = true;
 
-            if (!gameObject.activeInHierarchy)
+            if(!gameObject.activeInHierarchy)

```

Comment:
Can you please revert all the formatting changes in this and other files? I know it's a bit of a nitpick but there are a few things in your PR that break from the convention that's used throughout the rest of the codebase:
 - There should always be a space before the opening `(` after an `if`, `while`, `using`, `do`, or `for` statement, e.g. `if (SomeExpression())`
 - When breaking a long statement onto multiple lines, only indent one level - don't indent to make the code line up with something on the previous line. When breaking a function call onto multiple lines, the closing bracket should always be on the same indent level as the opening bracket, and if there are multiple arguments each argument should be on a separate line:
```cs
// Do this:
var monoBehaviourViewModel = cache.Components
    .FirstOrDefault(component =>
        component.GetType().ToString() == viewModelName
    );

var providedViewModel = cache.Components
    .Select(component => component.GetViewModelData())
    .Where(component => component != null)
    .FirstOrDefault(
    // etc...
```